### ISSUE TITLE
Move "Compact mode" and "Archived cards" toggles to the global settings in the top left

### ISSUE THREAD

**Issue by stefan-niedermann** [nextcloud/deck#2108]
Currently we have four buttons in the top right of each board:

![grafik](https://user-images.githubusercontent.com/4741199/86889983-edac3900-c0fc-11ea-885a-fd52e32ce196.png)

While "Filter" and "Sidebar" are contextually depending on the current board and are "resetted" when switchting the board, "Compact mode" and "Archived cards" are in a global context which means, their state is preserved when navigating through different boards.

I think the correct place for "Compact mode" and "Archived cards" is therefore in the global settings in the bottom left

![grafik](https://user-images.githubusercontent.com/4741199/86890149-2a783000-c0fd-11ea-84ad-06e73f55e8af.png)

## Considered alternatives

Make the context of "Compact mode" and "Archived cards" also depending on the current board and reset them when switching the board.

Maybe a mix of the two solutions would be the best:

1. Move "Compact mode" to the global settings as this is a setting which is probably only set once
2. Make context of "Archived cards" local, so it resets when switching the board - IMHO it doesn't make much sense to keep the state in another board

**Comment by juliushaertl** [comment]
I agree for the compact mode. The archived view will me migrated anyways to the board sidebar with #1930

**Comment by armaccloud** [comment]
@juliushaertl  @stefan-niedermann 
Why do you think the compact mode would probably be set once?
While using different boards, I often switch between compact and normal mode depending on the amount of cards that are present on the board. For large boards it works better to have the compact mode to remain overview, while in smaller boards it makes sense to see more details on cards. My preference would be to be able to set this setting per board and it to retain that setting.

**Comment by stefan-niedermann** [comment]
In this issue i just described the status quo - and according to the status quo the setting makes more sense in the bottom left settings. :man_shrugging:

**Comment by armaccloud** [comment]
Ah my bad! I thought you closed my issue #2127 as a duplicate from this one, but I see now that it was linked as an impact :)

**Comment by splashote** [comment]
Is this decided already?

As the 'archived cards'-button will move to the sidebar anyways, I'd prefer to stick to the 'compact mode' as one of three buttons.

I think of wanting to have a better overview on a deck and then switching back to the 'normal mode': It's either six 'clicks' or two.

**Comment by stefan-niedermann** [comment]
@splashote this issue is "exclusive or" to #2127 - we will either change the context *or* the position of the compact mode, because the current implementation is wrong. The solution has afaik not yet been decided finally by @juliushaertl
